### ISSUE TITLE
`typesof` not defined in `InteractiveUtils` while using Cthulhu

### ISSUE THREAD

**Issue by jishnub** [JuliaLang/julia#47606]
```julia
julia> using Cthulhu

julia> a = (; a = 1, b= 2)
(a = 1, b = 2)

julia> @descend a.a
ERROR: UndefVarError: typesof not defined
Stacktrace:
 [1] top-level scope
   @ ~/packages/julias/julia-1.8/share/julia/stdlib/v1.8/InteractiveUtils/src/macros.jl:81

julia> versioninfo()
Julia Version 1.8.2
Commit 36034abf260 (2022-09-29 15:21 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: 8 × 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-13.0.1 (ORCJIT, tigerlake)
  Threads: 1 on 8 virtual cores
Environment:
  LD_LIBRARY_PATH = :/usr/lib/x86_64-linux-gnu/gtk-3.0/modules
  JULIA_EDITOR = subl

(@v1.8) pkg> st -m Cthulhu
Status `~/.julia/environments/v1.8/Manifest.toml`
  [f68482b8] Cthulhu v2.7.6
```

**Comment by rgobbel** [comment]
Says above that this is fixed, but I'm still getting an error. In what version is this fiixed?

```Julia
julia> using Cthulhu

julia> a = (; a = 1, b= 2)
(a = 1, b = 2)

julia> @descend a.a
ERROR: UndefVarError: `typesof` not defined
Stacktrace:
 [1] top-level scope
   @ ~/.julia/juliaup/julia-1.9.0-rc2+0.aarch64.apple.darwin14/share/julia/stdlib/v1.9/InteractiveUtils/src/macros.jl:84

julia> versioninfo()
Julia Version 1.9.0-rc2
Commit 72aec423c2a (2023-04-01 10:41 UTC)
Platform Info:
  OS: macOS (arm64-apple-darwin21.3.0)
  CPU: 8 × Apple M2
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, apple-m1)
  Threads: 1 on 4 virtual cores
Environment:
  JULIA_IMAGE_THREADS = 1

(@v1.9) pkg> st -m Cthulhu
Status `~/.julia/environments/v1.9/Manifest.toml`
  [f68482b8] Cthulhu v2.8.11

```

**Comment by timholy** [comment]
Looks like that never got backported to 1.9. I just added that label, but meanwhile there's a workaround at https://github.com/JuliaDebug/Cthulhu.jl/pull/430 (released as Cthulhu 2.8.12).

**Comment by rgobbel** [comment]
Thanks. I verified that it's fixed in a custom build from the master branch.
